tests/lib/rules/no-jquery-methods.js

```diff
@@ -0,0 +1,100 @@
+const rule = require('../../../lib/rules/no-jquery-methods');
+const MESSAGE = rule.meta.message;
+const BLACKLISTMETHOD = rule.meta.blacklist[0];
+const RuleTester = require('eslint').RuleTester;
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
+  }
+});
+
+ruleTester.run('no-jquery-methods', rule, {
+  valid: [
+    {
+      code: `
+        export default Ember.Component({
+          init() {
+            Ember.$().notBlacklistedMethod();
+          }
+        });`
+    },
+    {
+      code: `
+        export default Ember.Component({
+          init() {
+            this.$().notBlacklistedMethod();
+          }
+        });`
+    },
+    {
+      code: `
+        export default Ember.Component({
+          init() {
+            const myJQueryObj = this.$();
+            myJQueryObj[${BLACKLISTMETHOD}]();      
```

Comment:
Ya I totally agree this is an issue. I think this is a massive slippery slope, and I for the most part what we have is addressing the bulk of what we need. 

We should merge this guy and address additional things as they crop up.